config/checkstyle_checks.xml

```diff
@@ -55,6 +55,9 @@
     <property name="offCommentFormat" value="CSOFF\: .*"/>
     <property name="onCommentFormat" value="CSON\: .*"/>
   </module>
+  <module name="SuppressWithNearbyPlainTextCommentFilter">
+    <property name="commentFormat" value="SUPPRESS CHECKSTYLE (\w+)"/>
```

Comment:
Make commentFormat = `<property name="message"`

And it reveals design problem, there is no comments or at least this filter does not care about comment and work on any text file.
We need to rename property to something like:
AnchorText or nearbyText.

Please add in instance of filter check name to suppress only specific problems

>name="SuppressWithNearbyPlainTextCommentFilter

It sounds like "Comment" is misleading word

Reply:
Maybe rename filter to just SuppressWithNearbyPlainTextFilter?

And yeah, nearbyText instead of commentFormat sounds more appropriate.

Reply:
nearbyText - good

Reply:
We already have https://checkstyle.org/config_filters.html#SuppressWithPlainTextCommentFilter

I think SuppressWithNearbyTextFilter is good

Reply:
Done